commons-executors/build.gradle

```diff
@@ -4,6 +4,8 @@ apply from: "../gradle/shared.gradle"
 apply plugin: 'java'
 apply plugin: 'eclipse'
 
+sourceCompatibility = 1.6
```

Comment:
We would also need `ideaSetModuleLevel(idea, 1.6)`, so that devs dont break 1.6 compatibility when developing in Intellij.